### DISCUSSION TITLE
Quesion about my logs

### DISCUSSION THREAD

**Discussion by gregeeh** [blakeblackshear/frigate#7248] [Ask A Question] [answered]
Hi all,

Regarding the logs below:
[https://pastebin.com/raw/nM1uUMMe](url)

I've noticed errors are happening every 7 days at 2:00AM local time.  There's nothing I've scheduled to happen at this time on the server containing the Frigate Docker Container.  Any ideas what it might be?

TIA

**Comment by NickM-27** [comment, marked as answer]
2 AM is the default time for cameras to restart, reolink also enables this by default. You should disable it in settings

**Reply by gregeeh** [reply]
Thanks, I would never of thought of that.
